src/components/MultipieceModal/MultipieceModal.test.js

```diff
@@ -0,0 +1,240 @@
+import React from 'react';
+import {
+  fireEvent,
+  render,
+  screen,
+} from '@testing-library/react';
+
+import '../../../test/jest/__mock__';
+
+import {
+  Modal,
+  ModalFooter,
+  Button,
+  KeyValue,
+} from '@folio/stripes/components';
+
+import {
+  componentPropsCheck,
+  getById,
+} from '../../../test/jest/helpers';
+
+import MultipieceModal from './MultipieceModal';
+
+describe('MultipieceModal', () => {
+  const testIds = {
+    multipieceModal: 'multipieceModal',
+    multipieceModalFooter: 'multipieceModalFooter',
+    multipieceModalConfirmButton: 'multipieceModalConfirmButton',
+    multipieceModalCancelButton: 'multipieceModalCancelButton',
+    numberOfPieces: 'numberOfPieces',
+    descriptionOfPieces: 'descriptionOfPieces',
+    numberOfMissingPieces: 'numberOfMissingPieces',
+    descriptionOfmissingPieces: 'descriptionOfmissingPieces',
+  };
+  const labelIds = {
+    multipieceModalLabel: 'ui-checkin.multipieceModal.label',
+    multipieceModalCheckIn: 'ui-checkin.multipieceModal.checkIn',
+    multipieceModalCancel: 'ui-checkin.multipieceModal.cancel',
+    multipieceModalMessage: 'ui-checkin.multipieceModal.message',
+    itemNumberOfPieces: 'ui-checkin.multipieceModal.item.numberOfPieces',
+    itemDescriptionOfPieces: 'ui-checkin.multipieceModal.item.descriptionOfPieces',
+    itemNumberOfMissingPieces: 'ui-checkin.multipieceModal.item.numberOfMissingPieces',
+    itemDescriptionOfMissingPieces: 'ui-checkin.multipieceModal.item.descriptionOfmissingPieces',
+  };
+  const testOpen = false;
+  const testItem = {
+    title: 'testTitle',
+    barcode: 'testBarcode',
+    materialType: {
+      name: 'testName',
+    },
+  };
+  const testOnClose = jest.fn();
+  const testOnConfirm = jest.fn();
+  const testDefaultProps = {
+    onClose: testOnClose,
+    onConfirm: testOnConfirm,
+    item: testItem,
+    open: testOpen,
+  };
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  describe('with default props', () => {
+    beforeEach(() => {
+      render(<MultipieceModal {...testDefaultProps} />);
+    });
+
+    it('should render multipiece "Modal" component', () => {
+      componentPropsCheck(Modal, testIds.multipieceModal, {
+        id: 'multipiece-modal',
+        size: 'small',
+        dismissible: true,
+        label: labelIds.multipieceModalLabel,
+      }, true);
+    });
+
+    it('should render multipiece "ModalFooter" component', () => {
+      componentPropsCheck(ModalFooter, testIds.multipieceModalFooter, {}, true);
```

Comment:
Does it have any sense to check props if you provide an empty object as expected props?
In this case I would say much better just check that `ModalFooter` was called.

`expect(ModalFooter).toBeCalled();`